src/Repository/TransactionRepository.php

```diff
@@ -44,4 +44,30 @@ public function findAllSuccessfulTransactionByEmail(string $emailAddress): array
             ->getResult()
         ;
     }
+
+    /**
+     * Total amount in cents
+     */
+    public function getTotalAmountInCentsByYearAndEmail(string $email, string $year = null): int
+    {
+        if (!$year) {
+            $year = date('Y');
```

Comment:
if it's not needed for you, then don't add it (YAGNI)
https://itexico.com/blog/bid/99765/software-development-kiss-yagni-dry-3-principles-to-simplify-your-life

Reply:
as u wish ^^, test in 2019 will be in fail ^^,

Reply:
What do you mean exactly?
Is there a way to fix this issue? By mocking the time for example?